gbfs.md

```diff
@@ -299,15 +299,15 @@ Describes bikes that are not at a station and are not currently in the middle of
 Field Name | Required | Type | Defines
 --|--|--|--
 `bikes` | Yes | Array | Array that contains one object per bike that is currently stopped as defined below.
-\-&nbsp;`bike_id` | Yes | ID | Identifier of a bike. *Beta (v2.0-RC):* Identifier of a bike, rotated to a random string, at minimum, after each trip to protect privacy. Note: Persistent bike_id, published publicly, could pose a threat to individual traveler privacy.
+\-&nbsp;`bike_id` | Yes | ID | Identifier of a bike, rotated to a random string, at minimum, after each trip to protect privacy *(as of v2.0-RC)*. Note: Persistent bike_id, published publicly, could pose a threat to individual traveler privacy.
```

Comment:
This table still seems to have a problem with wrapping.

Reply:
Yes, I saw that too. But I'm not sure how to fix it. It seems to be formatted the same as station_information.json, and that wraps...